src/translations/en/publication.json

```diff
@@ -31,14 +31,14 @@
     "date_published": "Date published",
     "description": "Description",
     "financed_by": "Financed by",
-    "npi_disciplines": "NPI disciplines",
+    "npi_disciplines": "Scientific fields Norwegian publication indicator",
```

Comment:
Dette var litt rart eller?

Reply:
Mangler kanskje en "for" i disse. kan høre med Hege